models/voteModel.js

```diff
@@ -0,0 +1,28 @@
+const { Schema, model } = require("mongoose");
+
+const voteSchema = Schema(
+  {
+    id: {
+      type: String,
+      required: true,
+      unique: true,
+    },
+    user_id: {
+      type: String,
+      required: true,
+    },
+    content_id: {
+      type: String,
+      required: true,
+    },
+    action: {
```

Comment:
you should be able to use enums here, like having just 2 available options, any other thing would be considered as invalid

Reply:
I checked oo, I don't know how to

Reply:
make it something like an options field, with only 2 choices

Reply:
I don't know if a boolean can work?

Reply:
> make it something like an options field, with only 2 choices

alright, let me work on this, I think I know what to do now